meta-lxatac-software/recipes-devtools/python/python3-labgrid/configuration.yaml

```diff
@@ -103,3 +103,17 @@ lxatac-usb-ports-p{{idx}}.{{if}}:
       '@ID_PATH': '{{sysfs}}:1.{{if}}'
 {% endfor %}
 {% endfor %}
+
+
+## Internal TFTP Server
+tftp:
+  TFTPProvider:
+    internal: "/srv/tftp/"
+    external: "/"
```

Comment:
Did you test this? The TFTP server needs be configured according to https://labgrid.readthedocs.io/en/latest/configuration.html#providers.